release/ray_release/scripts/build_pipeline.py

```diff
@@ -34,6 +34,19 @@
     type=str,
     help="File containing test configurations",
 )
+@click.option(
+    "--only-likely-failing-tests",
```

Comment:
maybe rename to `--likely-failing-tests-only`? or just `--likely-failing-only`?

I tried to think about a single word to replace "likely failing", but my English failed me.. so I asked chatgpt, and it gives me "precarious".

so maybe `--precarious-only` or `--precarious-tests-only`?

Reply:
Nice ;))

Reply:
and it also gives me 3 more options for backup:

<img width="803" alt="image" src="https://user-images.githubusercontent.com/95255098/232642560-21fbfe4e-f412-4332-af55-bae388cab781.png">

Reply:
lol i like 'tenuous', sound like an octopus